### ISSUE TITLE
Package 1.61.0 is not latest package from master

### ISSUE THREAD

**Issue by dominicusmento** [mjwsteenbergen/ApiLibs#33]
Hi,

can you please publish the latest package to nuget?
I'm doing some nuget version updates in https://github.com/tomidix/CSharpTradFriLibrary library but I see that your 1.61 is pretty old so I have problems and have to specify newest restSharp by myself..

I would be thankful if you could use major.minor.patch.build versioning schema, which corresponds to 
(breakingChanges.feature.fix.buildVersion).
For setting this in place I would recommend to use this SetVersion.ps1 script: https://github.com/Nodefusion/SetVersion
Basically, you put the script in your repo and then use it in yaml as task prior to build and it will change the build version number: 
`./SetVersion.ps1 -buildNumber $(Build.BuildNumber) -type csproj -filePath $(Build.SourcesDirectory)\ApiLibs\ApiLibs.csproj
`
Also I see that dll version (1.1.0) does not correspond to nuget, nuspec, version (1.61.0) which would be nice to be in sync, and I really don't know how that happened because nupkg should pick version from csproj..

**Comment by tomidix** [comment]
Any sign of having this sorted?

**Comment by mjwsteenbergen** [comment]
Hi!

Sorry that I haven't responded! Apparently I don't get notifications for created issues.

I have been busy updating the library. I would almost say its a version 2.0. It should make it a lot easier to incorporate the COAP library. This also means that there will be some changes that will have to be incorporated as well.

I have already created an updated fork and [am running a beta](https://github.com/mjwsteenbergen/CSharpTradFriLibrary) of it myself to make sure there are no regressions and fixing any bugs that I find along the way. Hopefully at the end of the week, if I have time, I will make a PR on the main repo and release the new version of ApiLibs, so that you will have to as little as possible

**Comment by dominicusmento** [comment]
Hi, I see that you have been active last couple of days..
New version sounds good. Hopefully you can publish it soon to nuget.org because the last one has that crucial bug where we have to specify restsharp in csproj along with apilibs..
